types/src/error.rs

```diff
@@ -58,4 +59,7 @@ pub enum DagError {
 
     #[error("Message {0} (round {1}) too old")]
     TooOld(Digest, Round),
+
+    #[error("Invalid epoch (expected {received}, received {received})")]
```

Comment:
This should probably be `"Invalid epoch (expected {expected}, received {received})"`

Reply:
good catch